bpf/bpf.go

```diff
@@ -2191,8 +2197,13 @@ func SupportsSockmap() error {
 	return nil
 }
 
+func GetExpectedVersionFromMap(distName string) *version.Version {
```

Comment:
Name of the function could be improved.  Right now it's too generic: "expected for what?" "which map?"  How about `MinKernelVersionForDistro()`